data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt

```diff
@@ -0,0 +1,41 @@
+mass limit -2sigma -1sigma median +1sigma +2sigma -nucl.unc. +nucl.unc
+9.00000e+00 2.43625e-41 3.74963e-41 6.22428e-41 1.17659e-40 1.79306e-40 1.42237e-40 4.09002e-41
```

Comment:
Hi, @ibles I think the limit is still missing here, when comparing to the arxiv version (https://arxiv.org/src/2207.03764v3/anc/Fig7_SpinDependentneutronLimitandSensitivity.txt) there is one column missing 

(limit at 4.8e-41 from that file)

Reply:
You are totally right! Sorry, I missed it. Let me fix that.

Reply:
Good to go. Thanks for catching that Knut!